db/migrate/20201104192101_add_contact_attempts_to_patients.rb

```diff
@@ -0,0 +1,21 @@
+class AddContactAttemptsToPatients < ActiveRecord::Migration[6.0]
+  def up
+    add_column :patients, :contact_attempts, :integer, default: 0
+
+    # populate :contact_attempts
+    execute <<-SQL.squish
+      UPDATE patients
+      INNER JOIN (
```

Comment:
This was slow locally so I have a feeling this could be a very slow migration on prod. @timwongj or @sgober, did either of you have a chance to test his on a high number of monitorees?

Reply:
hmm haven't tested it yet but can do, this is how we've been querying/populating new fields so I would be kind of surprised if this took significantly longer. I think it might be because we don't have an index on `history_type` which I think we should actually add, considering that we allow users to filter by this. @holmesie thoughts?

Reply:
I think that's a good idea. It may also be worth filtering out purged on this.

Reply:
I think it's fine to add that index and then this can be resolved.